dotgov-domains/current-federal.csv

```diff
@@ -294,123 +249,124 @@ UNNPP.GOV,Federal Agency,Department of Energy,West Mifflin,PA
-NGC.GOV,Federal Agency,Department of Health And Human Services,Rockville,MD
-NIH.GOV,Federal Agency,Department of Health And Human Services,Bethesda,MD
-NIHSENIORHEALTH.GOV,Federal Agency,Department of Health And Human Services,Bethesda,MD
-NIOSH.GOV,Federal Agency,Department of Health And Human Services,Atlanta,GA
-NLM.GOV,Federal Agency,Department of Health And Human Services,Bethesda,MD
-NNLM.GOV,Federal Agency,Department of Health And Human Services,Bethesda,MD
-OPIOIDS.GOV,Federal Agency,Department of Health And Human Services,Washington,DC
-ORGANDONOR.GOV,Federal Agency,Department of Health And Human Services,Rockville,MD
-PANDEMICFLU.GOV,Federal Agency,Department of Health And Human Services,Washington,DC
-PHE.GOV,Federal Agency,Department of Health And Human Services,Washington,DC
-PSC.GOV,Federal Agency,Department of Health And Human Services,Rockville,MD
-PUBMED.GOV,Federal Agency,Department of Health And Human Services,Bethesda,MD
-PUBMEDCENTRAL.GOV,Federal Agency,Department of Health And Human Services,Bethesda,MD
-QUIC.GOV,Federal Agency,Department of Health And Human Services,Rockville,MD
-RECOVERYMONTH.GOV,Federal Agency,Department of Health And Human Services,Rockville,MD
-SAFEYOUTH.GOV,Federal Agency,Department of Health And Human Services,Atlanta,GA
-SAMHSA.GOV,Federal Agency,Department of Health And Human Services,Rockville,MD
-SELECTAGENTS.GOV,Federal Agency,Department of Health And Human Services,Washington,DC
-SMOKEFREE.GOV,Federal Agency,Department of Health And Human Services,Rockville,MD
-STOPALCOHOLABUSE.GOV,Federal Agency,Department of Health And Human Services,Washington,DC
-STOPBULLYING.GOV,Federal Agency,Department of Health And Human Services,Washington,DC
-SURGEONGENERAL.GOV,Federal Agency,Department of Health And Human Services,Washington,DC
-THECOOLSPOT.GOV,Federal Agency,Department of Health And Human Services,Rockville,MD
-THEREALCOST.GOV,Federal Agency,Department of Health And Human Services,Washington,DC
-THISFREELIFE.GOV,Federal Agency,Department of Health And Human Services,Washington,DC
-TISSUEENGINEERING.GOV,Federal Agency,Department of Health And Human Services,Bethesda,MD
-TOBACCO.GOV,Federal Agency,Department of Health And Human Services,Washington,DC
-USABILITY.GOV,Federal Agency,Department of Health And Human Services,Washington,DC
-USBM.GOV,Federal Agency,Department of Health And Human Services,Atlanta,GA
-USPHS.GOV,Federal Agency,Department of Health And Human Services,Rockville,MD
-VACCINES.GOV,Federal Agency,Department of Health And Human Services,Washington,DC
-WHAGING.GOV,Federal Agency,Department of Health And Human Services,Washington,DC
-WHITEHOUSECONFERENCEONAGING.GOV,Federal Agency,Department of Health And Human Services,Washington,DC
-WOMENSHEALTH.GOV,Federal Agency,Department of Health And Human Services,Washington,DC
-YOUTH.GOV,Federal Agency,Department of Health And Human Services,Washington,DC
+ACF.GOV,Federal Agency,Department of Health and Human Services,Washington,DC

```

Comment:
What's the nature of the HHS diffs here? I can't tell from looking at them.

Reply:
"And" is no longer capitalized.

Reply:
Man, I looked at that over and over and didn't see it... good fix.